docs/scripts/install.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, path="/usr/local/bin"):
         self.bin_path = "%s/theme" % self.install_path
         self.arch = self.__getArch()
         print("Fetching release data")
-        self.release = json.loads(urllib.urlopen(Installer.LATEST_RELEASE_URL).read().decode("utf-8"))
+        self.release = json.loads(self.__req(Installer.LATEST_RELEASE_URL).decode("utf-8"))
```

Comment:
Would it be any better to use `from six.moves.urllib.request import urlopen` instead of `self.__req`?

Reply:
Will that work for both? Also what is `six.moves`

Reply:
Ah I found out! https://six.readthedocs.io/ This is great! Thanks @mllemango !!

Reply:
Yup! Works on my end. 
https://six.readthedocs.io/#module-six.moves.urllib.request `six.moves.urllib.request` wraps py2's urllib and py3's urllib.request. Prevents you from having to write that extra module.